pkg/oauth/oauth.go

```diff
@@ -0,0 +1,452 @@
+package oauth
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"net/http"
+	"os"
+	"path/filepath"
+	"time"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/pkg/browser"
+	"github.com/zalando/go-keyring"
+	"golang.org/x/oauth2"
+
+	"github.com/ankitpokhrel/jira-cli/internal/cmdutil"
+	"github.com/ankitpokhrel/jira-cli/pkg/utils"
+)
+
+const (
+	// JIRA OAuth2 endpoints.
+	jiraAuthURL            = "https://auth.atlassian.com/authorize"
+	jiraTokenURL           = "https://auth.atlassian.com/oauth/token"
+	accessibleResourcesURL = "https://api.atlassian.com/oauth/token/accessible-resources"
```

Comment:
Make use the same prefix as above


```suggestion
	jiraAccessibleResourcesURL = "https://api.atlassian.com/oauth/token/accessible-resources"
```